deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -0,0 +1,278 @@
+      [comet.address, arbitrumCOMPAddress]
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address, cometAdmin.address, rewards.address],
+        [0, 0, 0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+          'deployAndUpgradeTo(address,address)',
+          'setRewardConfig(address,address)'
+        ],
+        [setConfigurationCalldata, deployAndUpgradeToCalldata, setRewardConfigCalldata]
+      ]
+    );
+
+    const createRetryableTicketGasParams = await estimateL2Transaction(
+      {
+        from: applyL1ToL2Alias(timelock.address),
+        to: bridgeReceiver.address,
+        data: l2ProposalData
+      },
+      deploymentManager
+    );
+
+    const ENSResolver = await govDeploymentManager.existing('ENSResolver', ENSResolverAddress);
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain);
+    const arbitrumChainId = (await deploymentManager.hre.ethers.provider.getNetwork()).chainId.toString();
+    const newMarketObject = { baseSymbol: 'USDC.n', cometAddress: comet.address };
+    const officialMarketsJSON = JSON.parse(await ENSResolver.text(subdomainHash, ENSTextRecordKey));
+    if (officialMarketsJSON[arbitrumChainId]) {
+      officialMarketsJSON[arbitrumChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[arbitrumChainId] = [newMarketObject];
+    }
+
+    const mainnetActions = [
+      // 1. Set Comet configuration and deployAndUpgradeTo new Comet on Arbitrum.
+      {
+        contract: arbitrumInbox,
+        signature: 'createRetryableTicket(address,uint256,uint256,address,address,uint256,uint256,bytes)',
+        args: [
+          bridgeReceiver.address,                           // address to,
+          0,                                                // uint256 l2CallValue,
+          createRetryableTicketGasParams.maxSubmissionCost, // uint256 maxSubmissionCost,
+          refundAddress,                                    // address excessFeeRefundAddress,
+          refundAddress,                                    // address callValueRefundAddress,
+          createRetryableTicketGasParams.gasLimit,          // uint256 gasLimit,
+          createRetryableTicketGasParams.maxFeePerGas,      // uint256 maxFeePerGas,
+          l2ProposalData,                                   // bytes calldata data
+        ],
+        value: createRetryableTicketGasParams.deposit
+      },
+      // 2. Approve the COMP gateway to take Timelock's COMP for bridging
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [compGatewayAddress, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from mainnet to Arbitrum rewards
```

Comment:
Do we need to bridge more COMP to Arbitrum?

Seems like we should just turn the reward speed on the first market to 0.

Reply:
I see now.  

That make sense, we should just need to do it once and don't necessary do ti again for USDC native as long as reward contract has enough rewards token.

Reply:
Hmm not sure what happened.  
But after removing those unneeded and seems innocent steps, scenario liquidator bot started failing again :(

Reply:
Solved it,  by adding existing rewards contract to deploy script. :)